src/core/api/link_raw.hpp

```diff
@@ -96,7 +96,7 @@ class LinkRaw
      * This method invokes the mTransmitDoneCallback, if set.
      *
      */
-    void InvokeTransmitDone(otRadioFrame *aFrame, bool aFramePending, otError aError);
+    void InvokeTransmitDone(const otRadioFrame *aFrame, const otRadioFrame *aAckFrame, otError aError);
```

Comment:
Not necessarily in this PR, but we could improve the doxygen in this file (e.g., document parameters and add explicit `void`).